tests/configure_test.py

```diff
@@ -56,3 +56,37 @@ def test_app(self):
         expected["hello"] = "hi"
         self.run_app()
         self.assertEqual(expected, self.app.config)
+
+    def test_config_value_error(self):
+        config = {
+            "buckets": {
+                "myBucket": {
+                    "secretKey": "ffffffffffffffffffuuuuuuuuuuuuuu"
+                },
+                "myOtherBucket": {
+                    "accessKey": "imGaGaGonnaMakeYouSomeSoup",
```

Comment:
Don't be silly we got a job to do. No horsin' 'round. Lol I felt staccato was the way to go with this one.